fe/fe-core/src/main/java/com/starrocks/catalog/HiveTable.java

```diff
@@ -329,11 +330,27 @@ private void validate(Map<String, String> properties) throws DdlException {
             throw new DdlException("property " + HIVE_METASTORE_URIS + " or " + HIVE_RESOURCE + " must be set");
         }
 
+        // 1. check conflicts if both hive.metastore.uris and resource are set.
+        // 2. try to set proper resource if hive.metastore.uris is specified and hive.resource not.
+        HiveResource hiveResource = null;
+        HiveResource hiveResourceByUris = null;
+        HiveResource hiveResourceByName = null;
         if (!Strings.isNullOrEmpty(hiveMetastoreUris)) {
```

Comment:
I think it is better to remove the interface to create table with hive.metastore.uris.
Users can only create tables by specifying resource.

Reply:
Just ignore the hive.metastore.uris properties. And the resource must be specified.

Reply:
done, give a prompt here if hive.metastore.uris is specified.

@gengjun-git @wyb PTAL